recipes/kissplice/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: kissplice
+  version: '2.4.0.p1'
+
+source:
+  fn: kissplice-2.4.0-p1.tar.gz
+  url: "ftp://pbil.univ-lyon1.fr/pub/logiciel/kissplice/download/kissplice-2.4.0-p1.tar.gz"
+  md5: "36ef9e22a0607f0ab4516cc0d4cf9fea"
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - gcc   # [linux]
+    - llvm  # [osx]
+    - cmake
+    - zlib
+  run:
+    - libgcc  # [linux]
+    - zlib
```

Comment:
can you pin zlib against CONDA_ZLIB here.

Reply:
Done